packages/mongo/mongo_driver.js

```diff
@@ -992,6 +992,13 @@ MongoConnection.prototype._createSynchronousCursor = function(
     replaceTypes(cursorDescription.selector, replaceMeteorAtomWithMongo),
     cursorOptions.fields, mongoOptions);
 
+  if (typeof cursorOptions.maxTimeMS !== 'undefined') {
```

Comment:
This is going to be documented alongside other collection `find` options, like `pollingIntervalMs` and `pollingThrottleMs`. To keep naming consistent, please consider changing `cursorOptions.maxTimeMS` to `cursorOptions.maxTimeMs`.